lib/pure/random.nim

```diff
@@ -7,12 +7,71 @@
 #    distribution, for details about the copyright.
 #
 
-## Nim's standard random number generator. Based on
-## the ``xoroshiro128+`` (xor/rotate/shift/rotate) library.
+## Nim's standard random number generator.
+##
+## Its implementation is based on the ``xoroshiro128+``
+## (xor/rotate/shift/rotate) library.
 ## * More information: http://xoroshiro.di.unimi.it/
 ## * C implementation: http://xoroshiro.di.unimi.it/xoroshiro128plus.c
 ##
 ## **Do not use this module for cryptographic purposes!**
+##
+## Basic usage
+## ===========
+##
+## To get started, here are some examples:
+##
+## .. code-block::
+##
+##   import random
+##
+##   # Initializes the default random number generator
+##   # Without a call to randomize(), the same results will occur every time
```

Comment:
will -> would ?

Reply:
I think "will" is alright here since, from what I read, it seems to imply more certainty than "would", and it's certain what happens in this case.